crates/ruff_cli/src/commands.rs

```diff
@@ -303,22 +309,46 @@ pub fn rule(rule: &Rule, format: HelpFormat) -> Result<()> {
                 output.push_str("Message formats:");
                 for format in rule.message_formats() {
                     output.push('\n');
-                    output.push_str(&format!("* {}", format));
+                    output.push_str(&format!("* {format}"));
                 }
             }
-
-            writeln!(stdout, "{}", output)?;
         }
         HelpFormat::Json => {
-            writeln!(
-                stdout,
-                "{}",
-                serde_json::to_string_pretty(&Explanation {
-                    code: rule.code(),
-                    linter: linter.name(),
-                    summary: rule.message_formats()[0],
-                })?
-            )?;
+            output.push_str(&serde_json::to_string_pretty(&Explanation {
+                code: rule.code(),
+                linter: linter.name(),
+                summary: rule.message_formats()[0],
+            })?);
+        }
+    };
+
+    match format {
+        HelpFormat::Text => {
+            let parser = Parser::new_ext(
```

Comment:
Ironically I expected `HelpFormat::Text` to be the "raw text" and `HelpFormat::Markdown` to be the "rendered Markdown".

Reply:
On second thought, I do think it makes sense for `--format text` to render as plain-text, and `--format markdown` to render with `mdcat`. We could then make `--format markdown` the default. What do you think?

Reply:
Sure. I won't be able to get around to it for a few days, so feel free to rejig - should be straightforward🙂

Reply:
Okay cool, will do :)